cmd/executor/cmd/root.go

```diff
@@ -37,11 +37,13 @@ var (
 	opts     = &config.KanikoOptions{}
 	logLevel string
 	force    bool
+	cleanup  bool
```

Comment:
Instead of creating a new variable here, could we add this to [KanikoOptions](https://github.com/GoogleContainerTools/kaniko/blob/master/pkg/config/options.go#L20)? This should help with executing the comment below.